.pre-commit-config.yaml

```
@@ -0,0 +1,31 @@
+repos:
+ - repo: https://github.com/Quantco/pre-commit-mirrors-black
+   rev: 20.8b1
```

```suggestion
   rev: 22.3.0
```
to fix one error in the execution of the pre-commit.